frontend/src/app/api/service/cache/route.ts

```diff
@@ -0,0 +1,35 @@
+import cacheService from '@/service/cache/service';
+import type { NextRequest } from 'next/server';
+
+export async function GET(req: NextRequest) {
+  const key = req.nextUrl.searchParams.get('key');
+
+  if (!key) {
+    return Response.error();
+  }
+
+  try {
+    const data = await cacheService.getKey(key as string);
+    return Response.json({ data });
```

Comment:
This ts error appear. Is it fine?
`
Property 'json' does not exist on type '{ new (body?: BodyInit | null | undefined, init?: ResponseInit | undefined): Response; prototype: Response; error(): Response; redirect(url: string | URL, status?: number | undefined): Response; }'.
`

Reply:
I dont think there is a problem here